pdr_backend/trueval/trueval.py

```diff
@@ -41,14 +42,15 @@ def get_true_val(topic, initial_timestamp, end_timestamp):
 
     """
     try:
-        exchange_class = getattr(ccxt, topic["source"])
+        exchange_class = getattr(ccxt, topic.source)
         exchange_ccxt = exchange_class()
         price_initial = exchange_ccxt.fetch_ohlcv(
-            topic["pair"], "1m", since=initial_timestamp, limit=1
+            topic.pair, "1m", since=initial_timestamp, limit=1
         )
         price_end = exchange_ccxt.fetch_ohlcv(
-            topic["pair"], "1m", since=end_timestamp, limit=1
+            topic.pair, "1m", since=end_timestamp, limit=1
         )
         return (price_end[0][1] >= price_initial[0][1], False)
     except Exception as e:
+        print(f"Error getting trueval for {topic.pair} {e}")
         return (False, 0, True)
```

Comment:
(This is a comment for util/models.py, not trueval/trueval.py)

Why is util/models.py added? That's gonna be confusing with models/ directory.